### PR TITLE
[FLOC-1455] Document a ZFS compatible kernel on Fedora20 on AWS

### PR DESCRIPTION
**Pull request by wallrj**
Fixes https://clusterhq.atlassian.net/browse/FLOC-1455

Run Fedora20 AWS acceptance tests as follows:
  * `./admin/run-acceptance-tests --keep --distribution=fedora-20 --provider=aws --config-file=aws.config --branch=0.4.0pre1 --flocker-version=0.4.0pre1`

[Review on Reviewable](https://reviewable.io/reviews/clusterhq/flocker/1264)

### CODE REVIEW

File: docs/indepth/installation.rst [hunk 1]
```diff
@@ -215,7 +215,7 @@ Using Amazon Web Services
```
Comment (jongiddy) [inline docs/indepth/installation.rst]: Not part of your patch, but the step above is missing a `@` after `root`.

Also, previous to that, the code block starting with "You should..." is superfluous, repeating the subsequent step (and using a different example hostname to boot).

File: docs/indepth/installation.rst [hunk 2]
```diff
@@ -215,7 +215,7 @@ Using Amazon Web Services
 
    Kernels older than ``3.16.4`` have a bug that affects Flocker's use of ZFS.
```
Comment (jongiddy) [inline docs/indepth/installation.rst]: Expand this to identify that kernel must be after 3.16.4 and before 3.18.  Perhaps more like the Digital Ocean text?

File: docs/indepth/installation.rst (hunk 1 above)
Comment (jongiddy) [inline docs/indepth/installation.rst]: Not sure why we even bother enabling the root account. All the subsequent steps could use `sudo`, and some of them do

Comment (lukemarsden) [inline docs/indepth/installation.rst]: Flocker needs to be able to log in between nodes as root for p2p volume handoff.

File: flocker/provision/_install.py [hunk 3]
```diff
@@ -184,6 +176,18 @@ def task_install_digitalocean_kernel():
     ]
 
 
+def task_install_aws_fedora20_kernel():
+    """
+    Upgrade kernel on an AWS hosted Fedora20 virtual machine
```
Comment (jongiddy) [inline flocker/provision/_install.py]: Full-stop at end of Docstring

**Comment by jongiddy** [comment]
This is working OK, and can be merged.  The AWS section of the installation docs have terrible mixed formatting, but that is not part of this issue.

File: docs/indepth/installation.rst (hunk 1 above)
Comment (jongiddy) [inline docs/indepth/installation.rst]: OK, I'm currently editing the same page to add Ubuntu instructions, so I might drop that tid-bit in.

**Comment by adamtheturtle** [comment]
Closing, see parent issue.

**Pull request closed without merging**